server/middleware/setUpSentry.ts

```diff
@@ -1,22 +1,51 @@
-import express from 'express'
 import * as Sentry from '@sentry/node'
-
-import config from '../config'
+import * as Tracing from '@sentry/tracing'
+import express from 'express'
 import applicationVersion from '../applicationVersion'
+import config from '../config'
 
 export function setUpSentryRequestHandler(app: express.Express): void {
   if (config.sentry.dsn) {
+    // Prevent usernames which are PII from being sent to Sentry
+    // https://docs.sentry.io/platforms/python/guides/logging/data-management/sensitive-data/#examples
+    const anonymousId = Math.random().toString()
+    Sentry.setUser({ id: anonymousId, username: anonymousId })
```

Comment:
@bryangaledxw FYI AP have this CodeQL thing set up! Useful?

Reply:
Yes! Let's get that on our repo!

Reply:
I don't think I'm using Math.random in a security context as this error is is warning about.

>Using a cryptographically weak pseudo-random number generator to generate a security-sensitive value, such as a password, makes it easier for an attacker to predict the value.

I am using a pseudo-random number generator but it makes no difference if an attacker guesses what it is. The only way it's used is within Sentry to prevent real data coming through. We could use 'null' but I was worried it might incorrectly group transactions by the same username.

I reckon this is a 'false positive' alert @richpjames. Happy to change this to `null` or show you how this random value propagates?